ebusd-2.1.x/en/vaillant/_templates.csv

```diff
@@ -97,3 +97,12 @@ shortphone,STR:6,,,phone number
 mamode,UIN,0=circulation;1=dryer;2=zone;3=legiopump,,
 offmode,UIN,0=eco;1=night,,
 rcmode,UIN,0=off;1=modulating;2=thermostat,,
+slotIndex,UCH,,,array index for the zones. Array Index start at 0 for the first element
+slotCount,UCH,,,Number of total slots in the array. May be Zero.
+slotCountWeek,IGN:1;slotCount;slotCount;slotCount;slotCount;slotCount;slotCount;slotCount;IGN:1,,,Number of total slots in the array for a week. Mon-Sun
+slotTimeFrame,HTM;HTM,,,Timeslot from/until
+slotTemp,UIN,,,Numbers are 10 times higher than the actual temp (e.g. 225 = real 22.5)
```

Comment:
in this case you should use a divisor with value 10, same for DE I guess

Reply:
added the divisor. I hope it is correct now. Haven't used that feature yet. Looks better now:
```
localhost: r -c ctlv2 hwcTimer.Monday0
13:50;20:00;58.0
```